pysap/test/test_binding.py

```diff
@@ -42,17 +42,26 @@ def setUp(self):
     def test_wavelet_transformations(self):
         """ Test all the registered transformations.
         """
-        for image in self.images:
```

Comment:
Why renaming this parameter?

Reply:
to avoid confusion with the image copy of the image later on